crates/topos-tce-transport/src/lib.rs

```diff
@@ -5,8 +5,51 @@ use serde::{Deserialize, Serialize};
 use std::str::FromStr;
 use thiserror::Error;
 use topos_core::uci::{Certificate, CertificateId};
-use topos_crypto::messages::{Address, Signature, H160};
-use topos_p2p::PeerId;
+use topos_crypto::messages::Signature;
+use topos_crypto::messages::{Address, H160};
+
+#[derive(Debug, Error)]
+pub enum ValidatorIdConversionError {
+    #[error("Failed to parse address string as H160")]
+    ParseError,
+    #[error("Failed to convert byte array into H160")]
+    InvalidByteLength,
+}
+
+#[derive(Clone, Copy, Default, Debug, Serialize, Deserialize, Eq, PartialEq, Hash)]
+pub struct ValidatorId(H160);
```

Comment:
Maybe the `ValidatorId` should be part of the `topos-core` as it'll be used by other component not just the `broadcast`